### PR TITLE
[FIX] l10n_it_edi: document type misconfigured

### PR DESCRIPTION
**Pull request by malb-odoo**
When sending a document to the government, we have to put a certain document type. The document type are computed depending on what's on the lines of the move, the type of the move, etc

The mapping was wrongly done. For the moment when doing an invoice with a service, it will be mapped as a TD03 which was wrong. It should still be TD01. We misinterpreted the professional fees by just checking if there is more service than goods product but now we will add a tag that can be used on account for that.

task:4244462

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tsb-odoo** [comment]
@epictete LGTM

**Review by FlorianGilbert — APPROVED** [review]
Technically, it looks good to me.
Once you have the approval from the partner, you can r+ it yourself.

@robodoo delegate+

**Comment by malb-odoo** [comment]
@robodoo r+

File: addons/l10n_it_edi/models/account_move.py
Comment (epictete) [inline addons/l10n_it_edi/models/account_move.py:539]: Not sure having a space in a dict key is a good idea :nerd_face:

Comment (malb-odoo) [reply addons/l10n_it_edi/models/account_move.py:539]: Yep you are right :grimacing: will do a fix pr to be extra safe

**Pull request closed without merging**
